config/locales/en.yml

```diff
@@ -2695,6 +2696,7 @@ en:
     new:
       title: "Sign Up"
       tab_title: "Sign up"
+      login_to_authorize_html: "You need an OpenStreeMap account to access %{client_app_name}."
```

Comment:
Same here, maybe `Signup with OpenStreetMap to access...` or if we are going to have the same text in both places we should try and share one resource.